src/index.html

```diff
@@ -147,7 +163,54 @@ <h2 class="modal__title">New post</h2>
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script defer src="./scripts/validation.js"></script>
-    <script defer src="./scripts/index.js"></script>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-btn"></button>
+        <h2 class="modal__title">Edit profile avatar</h2>
+        <form id="edit-avatar-form" class="modal__form" novalidate>
+          <label for="profile-avatar-input" class="modal__label">
+            Profile Picture
+            <input
+              id="profile-avatar-input"
+              class="modal__input"
+              name="avatar"
+              placeholder="Paste the link to an image"
+              required
+            />
+            <span id="profile-avatar-input-error" class="modal__error"></span>
+          </label>
+          <button type="submit" class="modal__submit-btn">Save</button>
+        </form>
+      </div>
+    </div>
+
+    <div class="modal" id="delete-modal">
+      <div class="modal__delform">
+        <div class="modal__container">
+          <h2 class="modal__title modal__title-small">
+            Are you sure you want to delete this image?
+          </h2>
+          <form id="delete-form" class="modal__form modal__form-del">
+            <div class="modal__buttons">
+              <button
+                type="submit"
+                class="modal__submit-btn modal__submit-btn_del"
+              >
+                Delete
+              </button>
+              <button
+                type="button"
+                class="modal__submit-btn modal__submit-btn_cancel"
+              >
+                Cancel
+              </button>
+            </div>
+          </form>
+        </div>
+
+        <button type="button" class="modal__close-btn_del"></button>
```

Comment:
`modal__close-btn_del`

There is a BEM mistake


**Modifier** should be used only with its modifiable **basic** class: `block__element block__element_modifier-key_modifier-value`. Elements should get only **modifying styles** from modifier, and all other styles should be got from the basic class.


```html

        <button type="button" class="modal__close-btn modal__close-btn_del"></button>

```